README.md

```diff
@@ -118,6 +118,7 @@ This section is a great place to start if you want to get into improving OpenStr
 * [indoor=](https://indoorequal.org) - Map for viewing indoor data from OpenStreetMap.
 * [Qwant Maps](https://www.qwant.com/maps) - Modern map with POI search and routing functionality.
 * [F4map Demo](https://demo.f4map.com/) - 3D rendering demo of OpenStreetMap data.
+* [FacilMap](https://facilmap.org/) - Map with easy possibility to filter for POI types
```

Comment:
Two small things about this:
1. FacilMap is Open Source, so consider adding the repo link
2. Descriptions need to end with a '.'

```suggestion
* [FacilMap](https://facilmap.org/) - Map with easy possibility to filter for POI types. ([Source Code](https://github.com/FacilMap/facilmap))
```